src/_js/lib/Feedback.js

```diff
@@ -6,9 +8,29 @@ $(document).on('change', '[data-feedback-toggle]', function(event) {
   window.ra.event('docs.feedback-sent', {
     useful: parseInt($selected.val(), 10)
   });
+  obj.style.display="none";
```

Comment:
will this also dismiss it for other pages the user navigates to in the same session?

Reply:
Yes - for a period of 30 minutes